JamitLabs/JamitFoundation
# Add generic component for collection view's supplementary view

## Description

This MR includes following changes: 
* Implements a container view which can be used as a supplementary view in a collection view
* Adds shortcuts for registration and dequeuing supplementary view in runtime
* Adds demo interface demonstrating how a newly added container view can be used

## How to test it

1. Run the sample app
2. Select "CollectionView"
3. Observe the behavior
